src/MakePIAPortable/MakePIAPortable.cs

```diff
@@ -37,6 +37,47 @@ class Program
             { "System.ValueType", System_Runtime },
             { "System.Collections.Generic.IEnumerable", System_Runtime },
             { "System.Reflection.DefaultMemberAttribute", System_Runtime },
+            { "System.Reflection.AssemblyDelaySignAttribute", System_Runtime},
+            { "System.Runtime.CompilerServices.CompilationRelaxationsAttribute", System_Runtime },
+            { "System.Runtime.CompilerServices.RuntimeCompatibilityAttribute", System_Runtime },
+            { "System.Diagnostics.DebuggableAttribute", System_Runtime },
+            { "System.Reflection.AssemblyTitleAttribute", System_Runtime },
+            { "System.Runtime.Versioning.TargetFrameworkAttribute", System_Runtime },
+            { "System.Resources.SatelliteContractVersionAttribute", System_Runtime },
+            { "System.Reflection.AssemblyCompanyAttribute", System_Runtime },
+            { "System.Reflection.AssemblyConfigurationAttribute", System_Runtime },
+            { "System.Reflection.AssemblyCopyrightAttribute", System_Runtime },
+            { "System.Reflection.AssemblyFileVersionAttribute", System_Runtime },
+            { "System.Reflection.AssemblyInformationalVersionAttribute", System_Runtime },
+            { "System.Reflection.AssemblyProductAttribute", System_Runtime },
+            { "System.Runtime.InteropServices.ComVisibleAttribute", System_Runtime },
+            { "System.Runtime.InteropServices.ComSourceInterfacesAttribute", System_Runtime },
+            { "System.Runtime.InteropServices.ComEventInterfaceAttribute", System_Runtime },
+            { "System.Runtime.CompilerServices.CompilerGeneratedAttribute", System_Runtime },
+            { "System.MulticastDelegate", System_Runtime },
+            { "System.IAsyncResult", System_Runtime },
+            { "System.AsyncCallback", System_Runtime },
+            { "System.IDisposable", System_Runtime },
+            { "System.Runtime.InteropServices.ComTypes.IConnectionPointContainer", System_Runtime },
+            { "System.Collections.ArrayList", System_Runtime },
```

Comment:
>{ "System.Collections.ArrayList", System_Runtime }, [](http://example.com/codeflow?start=12&length=51)

This isn't in .net standard 1.3. Do we need it?

Reply:
I think we may need to strip out any `_EventProvider` classes to avoid this.

Reply:
Or maybe we could not process System.VisualStudio.Interop? I don't think we target anything against .net standard 1.3 anymore. So maybe everything would work if we just used the net45 version of the assembly? I am not sure if mono gets upset about COM interop stuff the same way that CoreCLR does.

Reply:
Actually, since our goal is to eventually move to CoreCLR, and there is probably a non-trivial amount of work to skip processing System.VisualStudio.Interop, unless it proves difficult to strip out the classes that use ArrayList that is probably the best option.

Reply:
Wrote code in the MakePIAPortable to skip the whole class if we see it contains `_EventProvider`. Let me know if this was what you were thinking.

Reply:
Or do we want to explicitly state which classes to skip.

Reply:
LGTM